### PR TITLE
Allow setting font path and font color by frequency

### LINKED ISSUE #10: Support color scales
Font color should either be a color or a function that return a color

### LINKED ISSUE #23: Allow setting font path
Currently hard-coded

### PR DESCRIPTION
**Pull request by naslundx**
Fixes #10 
Fixes #23 

Also added 'Color' as a type alias for tuple[int, int, int]

### CODE REVIEW

File: wrdcld/font.py [hunk 1]
```diff
@@ -6,14 +7,17 @@
...
 
 @dataclass(frozen=True)
 class FontWrapper:
-    path: Path = get_repo_root() / "fonts" / "OpenSans-Regular.ttf"
+    color_func: Callable[[float], Color]
+    path: Path
     size: int = 1
-    color: tuple[int, int, int] = (255, 255, 0)
+
+    def color(self, frequency: float) -> Color:
```
Comment (jimbarrett27) [inline wrdcld/font.py:19]: colour :angry:

Comment (jimbarrett27) [reply wrdcld/font.py:19]: (not actually bothered about it)

**Review by jimbarrett27 — APPROVED** [review]

File: wrdcld/font.py (hunk 1 above)
Comment (naslundx) [reply wrdcld/font.py:19]: First counterclockwise, now this :(

**Comment by naslundx** [comment]
So weird, I'm getting `wrdcld/util.py Imports are incorrectly sorted and/or formatted.` on CI. But there are only two imports, and I get it regardless of order

**Pull request merged**